### DISCUSSION TITLE
Moving to alpine instead of debian

### DISCUSSION THREAD

**Discussion by ameer1234567890** [netalertx/NetAlertX#455] [Ideas]
Are there any plans to move to alpine linux as a base image instead of debian. Alpine linux is super tiny and compared to debian. It would cut down on build time as well as deploy time, while also greatly reducing memory usage for users.

**Comment by jokob-sk** [comment]
Hey,

No plans, but happy if someone does a PR and tests that existing functionality is preserved.

j
